galley/cmd/galley/cmd/server.go

```diff
@@ -157,8 +157,8 @@ func serverCmd() *cobra.Command {
 	serverCmd.PersistentFlags().StringVar(&serverArgs.ValidationArgs.WebhookConfigFile,
 		"validation-webhook-config-file", "",
 		"File that contains k8s validatingwebhookconfiguration yaml. Required if enable-validation is true.")
-	serverCmd.PersistentFlags().UintVar(&serverArgs.ValidationArgs.Port, "validation-port", 443,
-		"HTTPS port of the validation service. Must be 443 if service has more than one port ")
+	serverCmd.PersistentFlags().UintVar(&serverArgs.ValidationArgs.Port, "validation-port", serverArgs.ValidationArgs.Port,
```

Comment:
`serverArgs.ValidationArgs.Port` defaults to port 443. This should be 9443 to match the targetPart change below.

Reply:
serverArgs.ValidationArgs.Port is set to 9443. so here we should use `serverArgs.ValidationArgs.Port` instead of hardcoded `443` here.

Reply:
Maybe I'm missing it, but where is the default port set to 9443? AFAICT, port 443 is still the default (see below).

https://github.com/istio/istio/blob/6ecc2c196d65474eb50981272cb0ba43be31e125/galley/pkg/server/settings/args.go#L152

https://github.com/istio/istio/blob/6ecc2c196d65474eb50981272cb0ba43be31e125/galley/pkg/crd/validation/webhook.go#L158-L171

Reply:
You saw that my previous PR - https://github.com/istio/istio/pull/15625 was reverted due to the issue in local test. I am working on resubmit the that PRs. I will notify you once the PR was merged. then you can help to review this PR. Thanks @ayj